### ISSUE TITLE
Lack of customization.

### ISSUE THREAD

**Issue by dmjohn0x** [kylepaulsen/RoR2-Original-Sound-Track#2]
Can we have a way to add several tracks to a specific map and then allow the plugin to randomly choose one of the tracks?

**Comment by kylepaulsen** [comment]
Yeah this is how the plugin currently works. If you edit settings.xml, you can define <song> tags to match music files to maps. For example:

```
<song
    name="song1.mp3"
    scenes="blackbeach"
    boss="false"
    volume="1"
/>
<song
    name="song2.mp3"
    scenes="blackbeach"
    boss="false"
    volume="1"
/>
```
In this case, the plugin will randomly choose between song1 and song2 when you are on the "blackbeach" map (aka Distant Roost - view README for map name <--> map id.).

Alternatively, if you don't have any <song> tags for a certain map, the plugin will notice this and randomly choose a music file among all the ones it found.

Hope this helps.
